bindings/java/native/build.rs

```diff
@@ -0,0 +1,103 @@
+use std::{
+    env, fs,
+    path::{Path, PathBuf},
+};
+
+use flapigen::{JavaConfig, JavaReachabilityFence, LanguageConfig};
+
+#[path = "src/foreign_types/attributes.rs"] mod attributes;
+
+fn main() {
+    env_logger::init();
+
+    let out_dir = env::var("OUT_DIR").unwrap();
+    let jni_c_headers_rs = Path::new(&out_dir).join("jni_c_header.rs");
+    gen_jni_bindings(&jni_c_headers_rs);
+    let have_java_9 = fs::read_to_string(&jni_c_headers_rs)
+        .unwrap()
+        .contains("JNI_VERSION_9");
+
+    let java_cfg = JavaConfig::new(
+        Path::new("src").join("main").join("java").join("org").join("iota").join("wallet"),
+        "org.iota.wallet".into(),
+    )
+    .use_reachability_fence(if have_java_9 {
+        JavaReachabilityFence::Std
+    } else {
+        JavaReachabilityFence::GenerateFence(8)
+    });
+
+    let in_src = Path::new("src").join("java_glue.rs.in");
+    let test_opt_rsc = Path::new("src").join("test_optional.rs.in");
+    let out_src = Path::new(&out_dir).join("java_glue.rs");
+    let swig_gen = flapigen::Generator::new(LanguageConfig::JavaConfig(java_cfg))
+        .rustfmt_bindings(true)
+        .remove_not_generated_files_from_output_directory(false)
+        .merge_type_map("chrono_support", include_str!("src/foreign_types/chrono_include.rs"))
+        .merge_type_map("foreign_types", include_str!("src/foreign_types/types.rs"))
+        .register_class_attribute_callback("PartialEq", attributes::class_partial_eq)
+        .register_class_attribute_callback("Display", attributes::class_to_string);
+        // TODO: Create a attribute for auto generation
```

Comment:
is this still a TODO?

Reply:
Some of these todos are for the dev of the generator, want me to remove em?